cmd.go

```diff
@@ -157,6 +157,12 @@ type Options struct {
 	// the real command. These functions can be used to customize the underlying
 	// os/exec.Cmd. For example, to set SysProcAttr.
 	BeforeExec []func(cmd *exec.Cmd)
+
+	// LineBufferSize sets the size of the OutputStream line buffer. The default
+	// value DEFAULT_LINE_BUFFER_SIZE is usually sufficient, but if
+	// ErrLineBufferOverflow errors occur, try increasing the size with this
+	// field.
+	LineBufferSize int
```

Comment:
Let's make it `unit` because it can't be negative (or zero). Then below, `if options.LineBufferSize > 0 {`.

Reply:
I'm unsure what you mean. `uint` instead or change the name of the field? 🙏

Reply:
Use type `uint` instead.

Reply:
I have updated the PR now. It does bring a, maybe only theoretical, implication. As `OutputStream.SetLineBufferSize` accept an `int` we need to cast it. Tihs could potentially bring an integer overflow as the range of `uint` is larger than `int` due to its unsigned nature. 🤔  WDYT, acceptable risk? Should we add a check?